Cargo.toml

```diff
@@ -36,11 +36,16 @@ tempfile = "3.8"
 arrow2 = "0.18"
 
 [workspace]
-members = ["gdal-sys"]
+members = [ "gdal-src","gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "54716dd8955d4f0561ce9bf8a83610b605e3c007" }
```

Comment:
This currently relays on a non-accepted patch to `proj-sys`: https://github.com/georust/proj/pull/190

Reply:
I pushed https://github.com/georust/gdal/pull/517/commits/27c2a5e396089f57bdd7a64f81c20183245b44bf which changes this to use the master branch of proj-sys instead as the relevant patch is now merged there. There is still no release that contains this change yet.

Reply:
Seems like we still need https://github.com/georust/proj/pull/192/files

Reply:
That's finally released with proj-sys 0.24 :tada: